ios/magnet.xcodeproj/xcshareddata/xcschemes/magnet.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0730"
+   LastUpgradeVersion = "0620"
```

Comment:
Changes are not necessary: http://stackoverflow.com/questions/16638322/what-is-xcscheme-file-what-is-it-purpose